lib/Extension/LanguageServerRename/Util/LocatedTextEditConverter.php

```diff
@@ -38,6 +40,11 @@ public function toWorkspaceEdit(LocatedTextEditsMap $map): WorkspaceEdit
                 )
             );
         }
+        $documentEdits[] = new RenameFile(
```

Comment:
this should be conditional i think, not all renames involve a file rename

Reply:
(if rename old URL != new URL?)

Reply:
Looking for this case I also added quick (currently silent i.e. no exception is thrown to catch by the handler and notify the client) return when the original name equals the new.